### PR TITLE
GraphiQL + Super-basic MisTickets

### PR DESCRIPTION
**Pull request by fforres**
Super-super-super Basic UI for "mis tickets"
Also, enables a graphiql interface to test :) 

image

### CODE REVIEW

File: app/api/gql/gql.ts [hunk 1]
```diff
@@ -43,6 +45,12 @@ export function graphql(source: string): unknown;
 export function graphql(
   source: "query FetchExampleEvents {\n  events {\n    id\n    description\n    community {\n      id\n      name\n    }\n    tags {\n      id\n      name\n      description\n    }\n  }\n}",
 ): (typeof documents)["query FetchExampleEvents {\n  events {\n    id\n    description\n    community {\n      id\n      name\n    }\n    tags {\n      id\n      name\n      description\n    }\n  }\n}"];
+/**
+ * The graphql function is used to parse GraphQL queries into a document that can be used by GraphQL clients.
+ */
+export function graphql(
+  source: "query misTickets($input: PaginatedInputMyTicketsSearchValues!) {\n  myTickets(input: $input) {\n    data {\n      approvalStatus\n      id\n      paymentStatus\n      redemptionStatus\n    }\n    pagination {\n      currentPage\n      pageSize\n      totalPages\n      totalRecords\n    }\n  }\n}",
```
Comment (hvergara) [inline app/api/gql/gql.ts:52]: are we going to name operations in english or spanish? we should choose one and follow that pattern going forward.

**Review by hvergara — APPROVED** [review]

File: app/api/gql/gql.ts (hunk 1 above)
Comment (fforres) [reply app/api/gql/gql.ts:52]: Agreed. English :)

Comment (fforres) [reply app/api/gql/gql.ts:52]: Will do a pass in the API to update naming

File: app/routes/_authenticated/graphiql/index.tsx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+import { Button } from "~/components/ui/button";
+import {
+  useIsAuthReady,
+  useIsLoggedIn,
+  useTokenRef,
+} from "~/utils/supabase/AuthProvider";
+import { useRefreshToken } from "~/utils/supabase/client";
+import { urls } from "~/utils/urls";
+
+const comunidades = `query TodasLasComunidades {
```
Comment (joseglego) [inline app/routes/_authenticated/graphiql/index.tsx:15]: The default language should be English.

**Review by joseglego — APPROVED** [review]
LGTM

**Pull request merged**